src/fides/api/ops/service/saas_request/override_implementations/domo_request_overrides.py

```diff
@@ -0,0 +1,73 @@
+import logging
+from json import dumps
+from typing import Any, Dict, List
+
+from requests import put
+
+from fides.api.ops.common_exceptions import (
+    ClientUnsuccessfulException,
+    ConnectionException,
+)
+from fides.api.ops.models.policy import Policy
+from fides.api.ops.models.privacy_request import PrivacyRequest
+from fides.api.ops.service.saas_request.saas_request_override_factory import (
+    SaaSRequestType,
+    register,
+)
+from fides.api.ops.util.saas_util import PRIVACY_REQUEST_ID
+from fides.ctl.core.config import get_config
+
+CONFIG = get_config()
+logger = logging.getLogger(__name__)
+
+
+@register("domo_user_update", [SaaSRequestType.UPDATE])
+def domo_user_update(
+    param_values_per_row: List[Dict[str, Any]],
+    policy: Policy,
+    privacy_request: PrivacyRequest,
+    secrets: Dict[str, Any],
+) -> int:
+    rows_updated = 0
+    # each update_params dict correspond to a record that needs to be updated
+    for row_param_values in param_values_per_row:
+        # get params to be used in update request
+        user_id = row_param_values.get("user_id")
+
+        # check if the privacy_request targeted emails for erasure,
+        # if so rewrite with a format that can be accepted by Domo
+        # regardless of the masking strategy in use
+        all_object_fields = row_param_values["all_object_fields"]
+
+        if "user.contact.email" in policy.get_erasure_target_categories():
```

Comment:
This is hard-coded at the moment to overwrite `email` and `alternateEmail` with a masked but properly formatted value. The ideal case would allow the user to specify a masking strategy at the field level to guarantee the fields can be masked even if the masking strategy defined in the policy is not compatible with the specific API.